spec/abci++/abci++_methods_002_draft.md

```diff
@@ -876,3 +896,23 @@ Most of the data structures used in ABCI are shared [common data structures](../
             * $t4$ "ADDED"
             * $t1$ "REMOVED" and `new_hashes` containing [$id(t3)$, $id(t4)$]
             * $t2$ "REMOVED" and `new_hashes` containing [$id(t3)$, $id(t4)$]
+
+### CanonicalVoteExtension
+
+>**TODO**: This protobuf message definition is not part of the ABCI++ interface, but rather belongs to the
+> Precommit message which is broadcast via P2P. So it is to be moved to the relevant section of the spec.
```

Comment:
Perhaps move this to the `encoding.md` file alongside `CanonicalVote`?

Reply:
I believe you mean `data_structures.md`. Yes, that's the idea. Once this design is accepted, I'm planning to do it in a further (small) PR.